modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -34,14 +34,16 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
-  profiles:
-  - AffinityAndTaints       <1>
-  - TopologyAndDuplicates   <2>
-  - LifecycleAndUtilization <3>
+  profileCustomizations:
+    podLifetime: 30m        <1>
```

Comment:
@damemi What would be a good example alternative pod lifetime to use in this example? Default is 24hours, but TE slides showed 5m. I wasn't sure whether customers were wanting longer or shorter times.

Reply:
any time is good, I think some wanted longer, so it might be good to call out the specific units that are available (days is not one of them, so an example of `48h` or something might be good to call that, it only goes up to `hours`)